_opcodes/op078-bg1.html

```diff
@@ -22,5 +22,15 @@
 <br/>7   Intelligence Penalty
 <br/>8   Wisdom Penalty
 <br/>9   Charisma Penalty
-<br/>'Damage' can be positive (penalty) or negative (bonus).
-<br/>Permanent timing modes require a duration for this effect to work.
+
+{% capture note %}
+'Damage' can be positive (penalty) or negative (bonus).
+{% endcapture %}
```

Comment:
Sometimes this is used with a note and sometimes not, so i changed all of them to be consistent across all opcodes